.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.4.0
+  hooks:
+    - id: trailing-whitespace
+    - id: end-of-file-fixer
```

Comment:
We shouldn't need to run styling lints through pre-commit-hooks, that should be handled by `ruff`.